connectors/rocketmq-replicator/src/main/java/org/apache/rocketmq/replicator/RmqSourceReplicator.java

```diff
@@ -117,10 +119,10 @@ public void start() {
         }
 
         buildRoute();
-        startListner();
+        startListener();
```

Comment:
startListener may not be needed, because the runtime has already done the load balancing logic, and put the load balancing queue in the context of each task

Reply:
> startListener may not be needed, because the runtime has already done the load balancing logic, and put the load balancing queue in the context of each task

I don't think it should be deleted, because the startListener method maintains changes in topic information, and the runtime maintains changes in task configuration and load balancing. Changes to a topic should not be maintained by the runtime :)

Reply:
I made a mistake. The source connecotr should be needed to sense the change of the source mq topic. It is not necessary on the sink side.

Reply:
> I made a mistake. The source connecotr should be needed to sense the change of the source mq topic. It is not necessary on the sink side.

this method is in the source connector, what do you mean by sink side? in addition, can you review the latest commits :)